### PR TITLE
[Doc] Add flink/spark connector rn

### PR DESCRIPTION
**Pull request by hellolilyliuyi**
Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:

- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.2
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

File: docs/ecosystem_release/flink_connector.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# Flink connector
+
+## **Notifications**
+
+**User guide:** [Load data using Flink connector](../loading/Flink-connector-starrocks.md)
```
Comment (banmoy) [inline docs/ecosystem_release/flink_connector.md:5]: should include unloading doc

File: docs/ecosystem_release/flink_connector.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+**Naming format of the JAR file:**
+
+- Flink 1.15 and later: `flink-connector-starrocks-${connector_version}_flink-${flink_version}.jar`
+- Prior to Flink 1.15: `flink-connector-starrocks-${connector_version}_flink-${flink_version}_${scala_version}.jar`
+
+**Methods to obtain the JAR file:**
+
+- Directly download the the Flink connector JAR file from the [Maven Central Repository](https://repo1.maven.org/maven2/com/starrocks).
+- Add the Flink connector as a dependency in your Maven project's `pom.xml` file and download it. For specific instructions, see [user guide](../loading/Flink-connector-starrocks#obtain-flink-connector).
```
Comment (banmoy) [inline docs/ecosystem_release/flink_connector.md:17]:
```suggestion
- Add the Flink connector as a dependency in your Maven project's `pom.xml` file and download it. For specific instructions, see [user guide](../loading/Flink-connector-starrocks.md#obtain-flink-connector).
```

File: docs/ecosystem_release/flink_connector.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+- Compile the source codes into Flink connector JAR file. For specific instructions, see [user guide](../loading/Flink-connector-starrocks#obtain-flink-connector).
```
Comment (banmoy) [inline docs/ecosystem_release/flink_connector.md:18]:
```suggestion
- Compile the source codes into Flink connector JAR file. For specific instructions, see [user guide](../loading/Flink-connector-starrocks.md#obtain-flink-connector).
```

File: docs/ecosystem_release/flink_connector.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+## **Release note**
+
+### 1.2
+
+**1.2.8**
+
+This release includes some improvements and bug fixes. The notable changes are as follows:
+
+- Support Flink 1.16 and 1.17.
+- Recommend to set `sink.label-prefix` when the sink is configured to guarantee the exactly-once semantics. For the specific instructions, see [Exactly Once](../loading/Flink-connector-starrocks#exactly-once).
```
Comment (banmoy) [inline docs/ecosystem_release/flink_connector.md:40]:
```suggestion
- Recommend to set `sink.label-prefix` when the sink is configured to guarantee the exactly-once semantics. For the specific instructions, see [Exactly Once](../loading/Flink-connector-starrocks.md#exactly-once).
```

File: docs/ecosystem_release/flink_connector.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+**Improvements**
+
+- Support to configure whether to use Stream Load transaction interface. [#228](https://github.com/StarRocks/starrocks-connector-for-apache-flink/pull/228)
```
Comment (banmoy) [inline docs/ecosystem_release/flink_connector.md:44]:
```suggestion
- Support to configure whether to use Stream Load transaction interface for at-least-once. [#228](https://github.com/StarRocks/starrocks-connector-for-apache-flink/pull/228)
```

File: docs/ecosystem_release/spark_connector.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
+# Spark connector
+
+## **Notifications**
+
+**User guide:** [Load data using Spark connector](../loading/Spark-connector-starrocks.md)
```
Comment (banmoy) [inline docs/ecosystem_release/spark_connector.md:5]: should also include unloading doc

File: docs/ecosystem_release/spark_connector.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+
+### 1.1
+
+**1.1.1**
+
+This release mainly includes some features and improvements for loading data to StarRocks.
+
+> **NOTICE**
+>
+> Take note of the some changes when you upgrade the Spark connector to this version. For details, see [Upgrade Spark connector](https://docs.starrocks.io/en-us/latest/loading/Spark-connector-starrocks#upgrade-from-version-110-to-111).
```
Comment (banmoy) [inline docs/ecosystem_release/spark_connector.md:34]: Should give the official url or relative doc url?

File: docs/ecosystem_release/spark_connector.md [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+- The sink supports retrying. [#61](https://github.com/StarRocks/starrocks-connector-for-apache-spark/pull/61)
+- Support to load data to BITMAP and HLL columns. [#67](https://github.com/StarRocks/starrocks-connector-for-apache-spark/pull/67)
+- Support to load ARRAY-type data. [#74](https://github.com/StarRocks/starrocks-connector-for-apache-spark/pull/74)
+- Support to flush according to the number of buffered rows. [#78](https://github.com/StarRocks/starrocks-connector-for-apache-spark/pull/78)
+
+**Improvements**
+
+- Remove useless dependency, and make the Spark connector JAR file lightweight. [#55](https://github.com/StarRocks/starrocks-connector-for-apache-spark/pull/55) [#57](https://github.com/StarRocks/starrocks-connector-for-apache-spark/pull/57)
+- Replace fastjson with jackson. [#58](https://github.com/StarRocks/starrocks-connector-for-apache-spark/pull/58)
+- Add missing the Apache license header. [#60](https://github.com/StarRocks/starrocks-connector-for-apache-spark/pull/60)
```
Comment (banmoy) [inline docs/ecosystem_release/spark_connector.md:47]:
```suggestion
- Add the missing Apache license header. [#60](https://github.com/StarRocks/starrocks-connector-for-apache-spark/pull/60)
```

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by banmoy — APPROVED** [review]

**Review by banmoy — APPROVED** [review]

**Review by amber-create — APPROVED** [review]

**Review by evelynzhaojie — APPROVED** [review]

**Comment by hellolilyliuyi** [comment]
@mergify backport  branch-3.2 branch-3.1 branch-3.0 branch-2.5

**Pull request merged**